imports/client/components/PuzzleListPage.tsx

```diff
@@ -502,6 +503,15 @@ const PuzzleListPage = (props: PuzzleListPageWithRouterParams) => {
             <StyledPuzzleListLinkLabel>Hunters</StyledPuzzleListLinkLabel>
           </StyledPuzzleListLinkAnchor>
         </StyledPuzzleListLink>
+        {/* Show firehose link only to operators */}
+        {tracker.canUpdate && (
+          <StyledPuzzleListLink>
+            <StyledPuzzleListLinkAnchor to={`/hunts/${props.match.params.huntId}/firehose`}>
+              <FontAwesomeIcon icon={faCloudShowersHeavy} />
```

Comment:
I'm fine with this choice, but did you also consider `faucet`?

Reply:
Faucet is good too; I didn't really have much of an opinion here.  Switched to faucet.